app/[locale]/what-is-ethereum-network/page.tsx

```diff
@@ -0,0 +1,761 @@
+                  >
+                    B
+                  </div>
+                  <div>
+                    <p className="font-bold">Barnabé Monnot</p>
+                    <p className="text-sm text-body-medium">
+                      Protocol Architecture, Ethereum Foundation
+                    </p>
+                  </div>
+                </div>
+              </Card>
+              <p>
+                {t.rich(
+                  "page-what-is-ethereum-network-staking-section-description-7",
+                  {
+                    validators: (chunks) => (
+                      <a href="https://beaconcha.in/charts/validators">
+                        {chunks}
+                      </a>
+                    ),
+                    staked_ether: (chunks) => (
+                      <a href="https://beaconcha.in/charts/staked_ether">
+                        {chunks}
+                      </a>
+                    ),
+                    strong: Strong,
+                  }
+                )}
+              </p>
+              <p>
+                {t.rich(
+                  "page-what-is-ethereum-network-staking-section-description-8",
+                  {
+                    strong: Strong,
+                  }
+                )}
+              </p>
+              <p>
+                {t(
+                  "page-what-is-ethereum-network-staking-section-description-9"
+                )}
+              </p>
+              <p>
+                {t(
+                  "page-what-is-ethereum-network-staking-section-description-10"
+                )}
+              </p>
+              <p>
+                {t.rich(
+                  "page-what-is-ethereum-network-staking-section-description-11",
+                  {
+                    lido: (chunks) => (
+                      <a href="https://lido.fi/ethereum">{chunks}</a>
+                    ),
+                    rocketpool: (chunks) => (
+                      <a href="https://www.rocketpool.net/">{chunks}</a>
+                    ),
+                    strong: Strong,
+                  }
+                )}
+              </p>
+              <div>
+                <LinkWithArrow href="/staking">
+                  {t(
+                    "page-what-is-ethereum-network-staking-section-description-12"
+                  )}
+                </LinkWithArrow>
+                <LinkWithArrow href="/run-a-node">
+                  {t(
+                    "page-what-is-ethereum-network-staking-section-description-13"
+                  )}
+                </LinkWithArrow>
+              </div>
+            </div>
+          </Section>
+
+          <Section
+            id={getId(tocItems[3].url)}
+            className="-scroll-mt-80 space-y-14"
+          >
+            <Image
+              src={layer2HubImg}
+              alt="Man and Dog Playing"
+              className="mx-auto"
+            />
```

Comment:
@corwintines although the width/height is included in the imported image, the `sizes` prop should be included to tell Next how big we actually need.


```tsx
// Import screens
import { screens } from "@/lib/utils/screen"

// Add to the two rectangular banner images
sizes={`(max-width: 832px) calc(100vw - 32px), (max-width: ${screens.lg}) 800px, (max-width: ${screens.xl}) calc(100vw - 480px), 800px`}
```

The two other images should have this as well but it would be adjusted a bit since they don't extend full width.